academic-services/knowledge-exploration-service/how-to-create-index-from-mag.md

```diff
@@ -2,27 +2,1410 @@
+        USING Outputters.Tsv(quoting : false);
+    
+        OUTPUT @fieldOfStudyJson
+        TO @Out_OutputPath + "/fieldsOfStudy/{PartitionNumber}.json"
+        USING Outputters.Tsv(quoting : false);
+    
+        OUTPUT @journalJson
+        TO @Out_OutputPath + "/journals/{PartitionNumber}.json"
+        USING Outputters.Tsv(quoting : false);
+    END;
+    ```
+
+1. Replace placeholder values in the script using the table below
+
+   |Value  |Description  | Example |
+   |---------|---------|------|
+   |**`@In_MagBlobAccount`** | The name of your Azure Storage account containing the Microsoft Academic Graph data set. | **mymagstore**
+   |**`@In_MagBlobContainer`** | The container name in your Azure Storage account containing the Microsoft Academic graph data set, usually in the form of **mag-YYYY-MM-DD**. | **mag-2020-02-07**
+   |**`@Out_OutputPath string`** | The entity output wasb path to where you'd like the formatted entities documents to go. | **wasb://makessubgraph@mymakesstore/2020-02-07/microsoft/entities**  |
+   |**`@Param_UseSubgraphForInstitution`** | Generates a subgraph containing data related to the specified institution. | **microsoft**|
+
+    > [!TIP]
+    > This tutorial uses Microsoft as an organization by default. You can target any organization by finding its NormalizedName in the Microsoft Academic Graph and then changing the value of the affiliationNormalizedNameFilter variable to said name.
+    >
+    > Please note that doing this may impact time estimates for script execution and index generation later on in this tutorial.
+
+5. Provide a **Job name**, change **AUs** to 50, and select **Submit**
+
+   ![Submit GenerateAzureSearchData job](media/how-to-create-index-create-job.png)
+
+The job should finish successfully in about 50 minutes
+
 ## Build index using MAKES command line tool
 
+Now that the data has been generated to build your index, we need to create some resources that the index will use and build the index using the kesm.exe tool.
+
 ### Create indexing resources
 
-### Submit index build job
+If you have not done so already, download the kesm.exe tool from your MAKES subscription.  See the [Create an API Instance](get-started-create-api-instances.md) for instructions on downloading the tool.
+
+1. Open a command prompt or terminal window and navigate to the folder where you have extracted the kesm.exe tool.
+
+1. Copy the following command to your command / terminal window:
+
+    ```cmd
+    kesm CreateIndexResources --IndexResourceName "<IndexResourceName>" --MakesPackage "<MakesPackageLocation>" --MakesIndexResourceConfigFilePath "<MakesIndexResourceConfigFilePath>"
+    ```
+
+1. Replace the command parameters with the values from the table below:
+
+    | Values | Description | Example |
+    |---------|---------|------|
+    |**`<IndexResourceName>`** | The name of the indexing resources you created above. | **makesindexres** |
+    |**`<MakesPackageLocation>`**
```

Comment:
To build the custom index, submit a job to ADLA using the kesm tool.  The tool uses the Azure resources created in the previous step to build the index and when finished will place the index in your storage account.